From: Antoine Musso <hashar@free.fr>
Date: Tue, 9 Jan 2018 14:44:06 +0100
Subject: WMF: pin requirements to what is deployed

Based on what has been previously grabbed via dh_virtualenv and is known
to work on production.

python-prettytable 0.7.2 is available in Jessie so we should use it.

Bug: T158243
Change-Id: I7446aa6441ed975c8ec1f35ba70085a1bb598bf9
---
 requirements.txt | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/requirements.txt b/requirements.txt
index 60968cb..b5030e2 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -8,11 +8,14 @@ GitPython==2.1.11
 smmap2==2.0.5
 gitdb2==2.0.5
 python-daemon>=2.0.4,<2.1.0
+lockfile==0.12.2
 extras
-statsd>=1.0.0,<3.0
+statsd==2.1.2
 voluptuous>=0.7,<0.10
-gear>=0.5.7,<1.0.0
-apscheduler>=3.0,<3.1.0
+futures==3.0.5
+gear==0.7.0
+apscheduler==3.0.6
 PrettyTable>=0.6,<0.8
 babel>=1.0
 six>=1.6.0
+tzlocal==1.2.2
